src/main/java/lotto/domain/RankLotto.java

```diff
@@ -0,0 +1,40 @@
+package lotto.domain;
+
+import java.util.Arrays;
+
+public enum RankLotto {
+
+    FIRST(6, 2000000000),
```

Comment:
```suggestion
    FIRST(6, 2_000_000_000),
```
이렇게 표기하면 단위를 알아보기 더 쉽더라구요!

Reply:
놓치고 있었네요